mcs/class/Xunit.NetCore.Extensions/Makefile

```diff
@@ -0,0 +1,17 @@
+thisdir = class/Xunit.NetCore.Extensions
```

Comment:
Ah, I see that e.g. ActiveIssueAttribute hardcodes the assembly name as Xunit.NetCore.Extensions so that won't work.